website/docs/scanformoduledependencies.md

```diff
@@ -0,0 +1,26 @@
+Enables the `Scan Sources for Module Dependencies` option for Visual Studio projects.
+
+```lua
+scanformoduledependencies "value"
+```
+If no value is set for a configuration, Visual Studio will default the option to `No`.
```

Comment:
I don't think we need to specify this.  It should be noted that this is the default IDE value, not `No` based on my previous comment.

Reply:
I've removed the line.